quark/mdk-2.0.q

```diff
@@ -292,38 +313,116 @@ namespace mdk {
             }
         }
 
+        Object get(String property) {
+            return _context.properties[property];
+        }
+
+        void set(String property, Object value) {
+            _context.properties[property] = value;
+        }
+
+        bool has(String property) {
+            return _context.properties.contains(property);
+        }
+
+        void route(String service, String version, String target, String targetVersion) {
+            Map<String,List<Map<String,String>>> routes;
```

Comment:
a) WTAF is this? Java?

b) Why are you going to a list so early? Wouldn't

```{
    service1: {
        "1.0.0": [ target1, targetVersion1 ],
        "1.0.1": [ target2, targetVersion2 ]
    },
    service2: { ... }
}
```

make more sense?